.travis.yml

```diff
@@ -41,6 +40,11 @@ install:
   - rm -rf eclipse
   - tar xzvf deps/eclipse-SDK-4.6.3-linux-gtk-x86_64.tar.gz eclipse
   - echo eclipseRoot.dir=$(pwd)/eclipse > eclipsePlugin/local.properties
+  - echo ossrhUsername=eller86 >> gradle.properties
```

Comment:
Are we certain that `gradle.properties` won't already exist?

Reply:
we're sure that we have it, because we commit it to this Git repo

https://github.com/spotbugs/spotbugs/blob/release-3.1/gradle.properties